plugins/view-resources/src/actions.ts

```diff
@@ -50,6 +50,21 @@ export function getSelection (focus: FocusSelection, selection: SelectionStore):
   return docs
 }
 
+export function queryAction (
+  query: LiveQuery,
+  client: Client,
+  doc: Doc | Doc[],
+  id: Ref<Action>,
+  callback: (action: Action | undefined) => void | Promise<void>,
+  derived: Ref<Class<Doc>> = core.class.Doc,
+  mode: ViewContextType = 'context'
+): boolean {
+  return query.query(view.class.Action, { 'context.mode': mode, _id: id }, async (actions) => {
```

Comment:
Why we need LiveQuery here, action is model object and shouldn't change without page reloading

Reply:
but action's `visibilityTester` is a function whose result might change without a page reload. for example, imagine having a ‘send to review’ action button on issue page that is only available for issues in status `in progress`. once your click this button, it should move the issue to different status, and the button should disappear

Reply:
Yes, but query doesn't know about it, and it will call own callback (which contain check visibility) only when this action modified (never I think)

Reply:
yeah, got it, you're right